docs/tutorials/how-to-deploy-a-serverless-function.md

```diff
@@ -0,0 +1,217 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+
+import TabItem from '@theme/TabItem';
+
+# How to Deploy a Serverless Function
+
+In this tutorial, you will learn how to deploy a function using Genezio, a serverless deployment platform that simplifies app management and reduces costs. We'll cover the benefits, answer common questions, and provide detailed deployment steps.
+
+**Do I Need to Modify My Function's Code?** Just make sure that your code is written according to the AWS Lambda Handler guide. [Read more](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html)
+
```

Comment:
This sentence is too tightly coupled with `AWS Lambda`.

To write a Genezio Function, the user has to set a `handler` and this is more or less the same idea for 
In the future, we plan to support easy migration from other cloud providers, too, so we don't want our user to be biased towards AWS technologies.

My suggestion is to either remove this sentence, or change it to:
```
**Do I need to modify my source code?** Make sure you export your function as a `handler` that will process events. Check out <section> to see a code snippet.